internal/pkg/cli/project_init.go

```diff
@@ -46,16 +46,25 @@ func NewInitProjectOpts() (*InitProjectOpts, error) {
 
 // Ask prompts the user for any required arguments that they didn't provide.
 func (opts *InitProjectOpts) Ask() error {
-	if opts.ProjectName != "" {
-		// User passed in a name as an argument.
+	// If there's a local project, we'll use that over anything else.
+	summary, _ := opts.ws.Summary()
+	if summary != nil {
```

Comment:
nit: I normally prefer the convention to check `err` instead of the returned value: if `err` is nil, then `summary` is valid. Otherwise `summary` is invalid